hax-lib/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "hax-lib"
+version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+edition.workspace = true
+repository.workspace = true
+readme.workspace = true
```

Comment:
And add a readme that says what this is doing.
In this case the `readme.workspace` will throw warnings because it won't use this one.